tools/generator/src/DTO/Target+LinkerFlags.swift

```diff
@@ -123,6 +123,7 @@ private func processLinkoptComponent(
             if let `extension` = filePath.path.extension,
                `extension` == "swiftmodule"
             {
+                // swiftlint:disable:next shorthand_operator
                 filePath = filePath + "\(swiftTriple).swiftmodule"
```

Comment:
Can we resolve this instead?

Reply:
It doesn't build

Reply:
I wonder if it needs `String()` on the rhs so it knows what the type is, or if we need to define another operator. Anyway, comment is fine for now.